arachne-sys-settings/src/main/java/com/odysseusinc/arachne/system/settings/service/impl/SystemSettingsServiceImpl.java

```diff
@@ -123,18 +132,36 @@ public boolean isSecuredSetting(SystemSetting systemSetting) {
         return systemSetting.getType() == SystemSettingType.password;
     }
 
-    private void updateProperty(List<SystemSetting> forUpdate, Map.Entry<Long, String> entry, SystemSetting systemProperty) {
+    private void updateProperty(List<SystemSetting> forUpdate, Map.Entry<Long, String> entry,
+                                SystemSetting systemProperty) throws BindException {
 
         String value = entry.getValue();
         if (value != null) {
             if (isSecuredSetting(systemProperty)) {
                 value = getEncryptedValue(value);
             }
+            if (PATTERN_SETTING_NAME.equals(systemProperty.getName())) {
```

Comment:
So are you going to add `if` for each new field, which requires such check? Why haven't you added new entry into SystemSettingType?

Reply:
if the second field with patterns appears - I will add a new type